buildScript/tasks.gincl

```diff
@@ -511,10 +511,11 @@ task dockerImage (dependsOn: loadConfig) {
         docker_tag = project.appConfigProps.docker_tag ?: docker_tag
         docker_registry = project.appConfigProps.docker_registry ?: docker_registry
         docker_registry = docker_registry == '' || docker_registry.endsWith('/') ? docker_registry : docker_registry + '/'
+        def docker_build_args = "--build-arg IMAGE_NAME=${docker_registry}${docker_repo} --build-arg APP_NAME=${project['app-name']}"
```

Comment:
Another approach to passing in an APP_NAME is to loop through workarea.  This has the advantage of supporting multi apps if we should deploy them that way.
I don't feel strongly about it.  You can decide.